ScipDotnet/ScipProjectIndexer.cs

```diff
@@ -21,6 +21,10 @@ public ScipProjectIndexer(ILogger<ScipProjectIndexer> logger) =>
     private void Restore(IndexCommandOptions options, FileInfo project)
     {
         var arguments = project.Extension.Equals(".sln") ? $"restore {project.FullName} /p:EnableWindowsTargeting=true" : "restore /p:EnableWindowsTargeting=true";
+        if (options.DotnetNugetConfigFile != null)
+        {
+            arguments += $" --configfile {options.DotnetNugetConfigFile.FullName}";
```

Comment:
Will this work correctly even if the path has spaces in it?

Reply:
Good catch, I will update to quote it